src/EditorFeatures/Core/Diagnostics/AbstractAggregateDiagnosticsTaggerProvider.SingleDiagnosticKindTaggerProvider.cs

```diff
@@ -115,13 +133,16 @@ protected internal virtual ImmutableArray<DiagnosticDataLocation> GetLocationsTo
             try
             {
                 var diagnostics = await _analyzerService.GetDiagnosticsForSpanAsync(
-                    document, range: null, cancellationToken: cancellationToken).ConfigureAwait(false);
+                    document,
+                    spanToTag.SnapshotSpan.Span.ToTextSpan(),
+                    diagnosticKinds: _diagnosticKind,
```

Comment:
Nit: rename the parameter to `diagnosticKind`

Reply:
done.